ap2/pairing/hap.py

```diff
@@ -192,11 +192,13 @@ def pair_list(self, req):
         res = [
             Tlv8.Tag.STATE, PairingState.M2
         ]
+        # TODO: 5.12.2 2. Verify that the controlle rsending the request has the admin bit set in the local pairings
+        #  list
         count = 0
         for file in os.listdir("./pairings/"):
             if file.endswith(".hap.pub"):
                 if count > 0:
-                    res.extend([Tlv8.Tag.SEPARATOR, None])
+                    res.extend([Tlv8.Tag.SEPARATOR, b'\x00'])
```

Comment:
Table 5-6:

Type | Name | Format | Description
-|-|-|-
0xFF | kTLVType_Separator | null | Zero-length TLV that separates different TLVs in a list.

Reply:
I've not seen any issues with it being `b'\x00'` - I assume it just skips over this value, though I am positive it **needs** something, even if it's null, because otherwise Tlv8.encode/decode don't work.

Reply:
I think problems may arise later when it comes to listing pairings, where multiple entries are separated by said Tag. Yeah - try with `b''` first, and put a code comment there that SEP should be null... then we can find it quickly later.

Reply:
Actually, does it work with just e.g. `res.extend([Tlv8.Tag.SEPARATOR])` ?

Reply:
Sorted.